usefulUtilities/modelJSONCleanup/README.md

```diff
@@ -0,0 +1,18 @@
+# Description
+
+Node script that can be used to help cleanup models.json
+
+# Use
+
+- Get an archive of the domain you would like
+- Extract the zip
+- Extract the models gzip file
+- Duplicate the generic js filter file for the specific domain
+- In the JS file add the path to the models.json and the new models.json
+- Follow the comment instructions to tweak to your needs
```

Comment:
which comment instructions?

Reply:
I was referring to the comments in the actual js file